doc/guides/backporting-to-release-lines.md

```diff
@@ -0,0 +1,81 @@
+
+## Staging branches
+
+Each release line has a staging branch that the releaser will use a scratch pad
+while preparing a release. The branch name is formatted as follows:
+`vn.x-staging` where `n` is the major release number.
+
+### Active staging branches
+
+| Release Line | Staging Branch |
+| ------------ | -------------- |
+| `v7.x`       | `v7.x-staging` |
+| `v6.x`       | `v6.x-staging` |
+| `v4.x`       | `v4.x-staging` |
+
+## What needs to be backported?
+
+When a release is being prepared, the releaser attempts to cherry-pick a
+certain set of commits from the master branch to the release staging branch.
+If a cherry-pick does not landing cleanly on the staging branch, the releaser
+will mark the pull request with a particular label for that release line,
+specifying to our tooling that this pull request should not be included. The
+releaser will then add a comment that a backport is needed.
+
+## What can be backported?
+
+The "Current" release line (currently v7.x) is much more lenient in what can
+be landed. Our LTS release lines (currently v4.x and v6.x) require that commits
+live in a Current release for at least 2 weeks before backporting. Please see
+the [`LTS Plan`][] for more information. After that time, if the commit can be
+cherry-picked cleanly from master, then nothing needs to be done. If not, a
+backport pull request will need to be submitted.
+
+## How to submit a backport pull request?
+
+For these steps, let's assume that a backport is needed for the v7.x release
+line. All commands will use the v7.x-staging branch as the target branch.
+In order to submit a backport pull request to another branch, simply replace
+that with the staging branch for the targeted release line.
+
+* Checkout the staging branch for the targeted release line
+* Make sure that the local staging branch is up to date with the remote
+* Create a new branch off of the staging branch
+
+```shell
+# Assuming your fork of Node.js is checked out in $NODE_DIR,
+# the origin remote points to your fork, and the upstream remote points
+# to git://github.com/nodejs/node
+cd $NODE_DIR
+git checkout v7.x-staging
+git pull -r upstream v7.x-staging
+# We want to backport pr #10157
+git checkout -b backport-10157
+```
+
+*Note: the branch name does not have to have the pull request number in it.*
+
+* After creating the branch, apply the changes to the branch.
+* The commit message should be identical to the commit message on the master
+  branch, unless the commit has to be different due to dependencies that are
+  not present in the targeted release line.
+  * The review metadata from the original commit should also be preserved in
+    backport commits.
+* Push the changes to your fork and open a pull request.
+* Be sure to target the `v7.x-staging` branch in the pull request.
+
+### Helpful Hints
+
+* Please include the backport target in the pull request title.
+  * Ex. `process: improve performance of nextTick (v4.x backport)`
```

Comment:
Instead of an example, I suggest we be prescriptive. I've been doing "v4.x backport: proces: improve...", I thought that was requested, but I'm happy to do above instead.